### PR TITLE
feat: allow to patch MFE caddyfile

### PR DESCRIPTION
**Pull request by snglth**
Comes in hand if you want to add basicauth

### CODE REVIEW

File: tutormfe/templates/mfe/apps/mfe/Caddyfile [hunk 1]
```diff
@@ -12,6 +12,8 @@
         }
     }
 
+    {{ patch("caddyfile-mfe") | indent(4) }}
```
Comment (regisb) [inline tutormfe/templates/mfe/apps/mfe/Caddyfile:15]: Please rename to "mfe-caddyfile".

**Review by regisb — CHANGES_REQUESTED** [review]
Thanks for the PR! Can you please add a changelog entry with `scriv create`?

File: tutormfe/templates/mfe/apps/mfe/Caddyfile (hunk 1 above)
Comment (snglth) [reply tutormfe/templates/mfe/apps/mfe/Caddyfile]: Done

**Comment by snglth** [comment]
Changelog entry created. Is it possible to also get it into 15.x version of tutor-mfe?

**Review by regisb — APPROVED** [review]
LGTM! FYI @arbrandes I let you check out this PR and merge/comment.

**Comment by regisb** [comment]
> Changelog entry created. Is it possible to also get it into 15.x version of tutor-mfe?

Unfortunately no, as we do not maintain older releases of Tutor and its plugins. You will have to maintain a temporary fork of tutor v15 yourself.

**Review by arbrandes — APPROVED** [review]
LGTM!

**Pull request merged**